components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -542,62 +545,37 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
             log.debug(String.format("Listing users by filter: %s %s %s", node.getAttributeValue(), node.getOperation(),
                     node.getValue()));
```

Comment:
Can a node take the null value?

Reply:
No, because the node is verified before passing to this API.